packages/nux/src/components/inline-tip/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { Notice } from '@wordpress/components';
+import { compose } from '@wordpress/compose';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+export function InlineTip( { isVisible, className, onRemove, children } ) {
+	if ( ! isVisible ) {
+		return null;
+	}
+
+	return (
+		<Notice className={ className } onRemove={ onRemove }>
+			{ children }
+		</Notice>
+	);
+}
+
+export default compose(
+	withSelect( ( select, { tipId } ) => ( {
+		isVisible: select( 'core/nux' ).isTipVisible( tipId ),
+	} ) ),
+	withDispatch( ( dispatch, { tipId } ) => {
+		const { disableTips, dismissTip } = dispatch( 'core/nux' );
+
+		return {
+			onRemove() {
+				// Disable reason: We don't yet have a <Confirm> component. One day!
+				// eslint-disable-next-line no-alert
+				if ( window.confirm( __( 'Would you like to disable tips like these in the future? ' ) ) ) {
```

Comment:
This behaviour of the tip disappearing when pressing cancel felt really unusual. Usually cancel is the safe option that has no effect whatsoever.

I think this needs some thought on the best way to handle disabling all tips.

Some thoughts.
- Rather than a second step through a modal, perhaps a better option would be to replace the current tip notice with another one with the message 'Remove all tips? Yes | No'. It's the same number of steps and the user is already looking at this part of the screen.
- It might be ok to just have `onRemove` dismiss only the single tip.
- Third option, which I think is best, is to show a 'Hide all messages like this' button in every tip.

Reply:
Yeah, I agree that the Cancel/Ok options here felt weird. In the original comps, we spelled out the choices a little more clearly: 

<img width="462" alt="Screen Shot 2019-07-15 at 8 03 47 AM" src="https://user-images.githubusercontent.com/1202812/61214892-2fb3c200-a6d7-11e9-8d0b-3b5db3092405.png">

I don't know if we can do that with the standard browser confirm modal though. If #16583 moves forward, we can just change the wording here. Otherwise, one of your other bullet point options is probably cleaner than what we have.

Reply:
I'll have a play with making our own custom `Modal` work. (See https://github.com/WordPress/gutenberg/pull/16582/files#r303307801 for the problem I was running int.) I agree it's the ideal user experience!

Reply:
Does the modal appear for every tip? If I'm a new user making my first journey through the editor, dismissing these tips one at a time, having to click 'No' every time would be pretty frustrating. Granted there are only two tips at the moment, so maybe not a huge issue blocking this PR.

Reply:
> Does the modal appear for every tip?

Should be just the first one. If you don't dismiss all tips from that first modal, you'll need to do it from within the Settings panel instead.

Reply:
👌 I made it so that the confirmation will only show once in 74e32052ab64e5fa20b3c470bddc0c0e3232126c.